src/main/plugins/browser-bookmarks-plugin/google-chrome-bookmark-repository.ts

```diff
@@ -27,7 +27,13 @@ export class GoogleChromeBookmarkRepository implements BrowserBookmarkRepository
                         FileHelpers.readFile(this.bookmarkFilePath)
                             .then((data) => {
                                 const jsonParsed = JSON.parse(data);
-                                const bookmarks: BrowserBookmark[] = this.getBookmarksFromObject(jsonParsed.roots.bookmark_bar);
+                                const rootsCombined = Object.values(jsonParsed.roots).reduce((combined: any, value: any) => {
+                                    if (value && value.children) {
+                                        combined.children = combined.children.concat(value.children)
+                                    }
+                                    return combined
+                                }, { children: [] } as object)
```

Comment:
Is this casting `as object` really needed?

Reply:
No, it doesn't look like it. I'm new to TypeScript. Updated the branch.